src/main/java/jp/ac/titech/c/se/stein/core/RepositoryRewriter.java

```diff
@@ -79,6 +69,18 @@
     @Option(names = "--extra-attributes", description = "rewrite encoding and signature in commits", order = Config.MIDDLE)
     protected boolean isRewritingExtraAttributes = false;
 
+    enum CacheLevel {
+        Blob, Tree, Commit
```

Comment:
Shall we use lowercase entries (`blob`, `tree`, `commit`) to make the parameter option `--cache-level` specifiable as a lowercase?